pages.ko/common/bzip2.md

```diff
@@ -14,3 +14,19 @@
 - 파일을 표준 출력으로 압축해제:
 
 `bzip2 -dc {{경로/압축된_.bz2파일}}`
+
+- 압축된 파일 내 각 파일의 무결성 테스트:
+
+`bzip2 --test {{path/to/compressed_file.bz2}}`
```

Comment:
Thanks for your contribution, can you translate the path placeholders too?

Feel free to refer to https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md

Reply:
@kbdharun 
I've applied those changes and also fixed some other awkward wording. Thanks!